### DISCUSSION TITLE
[이지어드민] 도착보장 주문 배송중 변경 문의

### DISCUSSION THREAD

**Discussion by park-junkyu0** [commerce-api-naver/commerce-solutions#45] [묻고 답하기]
안녕하세요, 이지어드민 개발팀 박준규입니다.

이전에 도착보장/선물하기 주문건이 수령자 주소 입력 시 일반 배송으로 전환된다고 하셨었는데요,
수령자 주소를 입력했을 때 판매처 내에서 주문 상태가 **배송중**으로 변경되는 부분은 따로 설정을 할 수 있는 건지 확인 부탁 드립니다.

도착보장 + 선물하기 주문 > 수령자가 배송지 입력 > 일반배송 + 배송중으로 변경되고 있다고 합니다.

이전 다른 고객사에선 배송지를 입력해도 결제완료로 조회가 됐었는데, 해당 고객사에선 배송중으로 변경된다고 하셔서요.

해당 내용 확인 부탁드립니다, 감사합니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

#### 1. 도착보장 주문건의 선물하기는 '일반배송' 주문으로 전환됩니다.
선물하기 주문건은 선물 수취자의 수락 시점에 배송지 정보를 입력합니다.
따라서 주문건의 결제 당시(선물을 보낼 당시) 도착보장일을 예측할 수 없기에 도착보장 선물하기 주문건은 일반배송으로 전환되어 처리됩니다.
- 관련한 스마트스토어센터 자주 묻는 질문: https://help.sell.smartstore.naver.com/faq/content.help?faqId=14918

#### 2. 도착보장 선물하기 주문건의 선물 수락 시 배송중 전환
네이버도착보장 주문건은 상품에 설정된 NFA물류사가 주문건을 수집하여 출고 처리를 진행합니다.
그렇기에 네이버도착보장 설정 상품을 선물하기로 주문하는 경우 아래와 같은 흐름으로 주문 처리가 진행됩니다.
1. 선물 수취자가 선물 수락을 완료
2. 설정된 NFA물류사가 대상 주문 정보를 수집
3. 대상 NFA물류사가 출고 관련 작업 처리와 주문 처리(발주 확인 또는 발송 처리)

따라서 이러한 경우 판매자 분께서는 이중출고의 위험이 있어 별도로 주문 처리를 진행하지 않는 방향을 권장합니다.
